File: backend/TrainerPart/TrainerPart/Models/MemberProgress.cs
using System;
using System.Collections.Generic;

namespace TrainerPart.Models;

public partial class MemberProgress
{
    public int ProgressId { get; set; }

    public int Mid { get; set; }

    public double Weight { get; set; }

    public double Bmi { get; set; }

    public DateTime RecordedDate { get; set; }

    public string? Remark { get; set; }

    public virtual Member MidNavigation { get; set; } = null!;
}

File: backend/member-service/src/main/java/com/flexify/member/entities/MemberProgress.java
package com.flexify.member.entities;

import java.time.LocalDateTime;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter

@Entity
@Table(name = "member_progress")
public class MemberProgress {
	
	

	    @Id
	    @GeneratedValue(strategy = GenerationType.IDENTITY)
	    private Integer progressId;

	    private Integer mid;

	    private double weight;
	    private double bmi;

	    private LocalDateTime recordedDate;

	    private String remark;
	}


